client/js/tracking.js

```diff
@@ -11,9 +11,19 @@ export default {
       const componentName = component.getAttribute('data-component-name');
       const componentId = component.getAttribute('data-component-id');
       const clickedProperties = JSON.parse(clicked.getAttribute('data-track-click'));
-      const properties = Object.assign({}, clickedProperties, {componentId, action: 'click'});
-
-      trackEvent({name: componentName, properties});
+      const properties = Object.assign(
+        {},
+        clickedProperties,
+        {componentId, action: 'click'}
+      );
+      const ariaExpanded = clicked.getAttribute('aria-expanded');
+      const propertiesWithAria = ariaExpanded ? Object.assign(
+        {},
+        properties,
+        {ariaExpanded}
```

Comment:
> Alternatively this logic could go in the show-hide component

[That's how we're dealing with the dynamic tracking of the content slider](https://github.com/wellcometrust/wellcomecollection.org/blob/master/client/js/components/content-slider.js#L310-L322) - we'd just need to make sure we also sending over other information to be able to differentiate where these components are.